.github/workflows/ci-fmudataio.yml

```diff
@@ -42,7 +42,12 @@ jobs:
             "fmu-sumo-uploader @ git+https://github.com/equinor/fmu-sumo-uploader.git"
 
       - name: Full test
-        run: pytest -v --cov --cov-report term-missing
+        run: |
+          if [[ "${{ github.event.pull_request.base.ref }}" == "staging" ]]; then
+            pytest -n auto -v --cov --cov-report term-missing --prod
+          else
+            pytest -n auto -v --cov --cov-report term-missing
+          fi
```

Comment:
Tests don't actually run on the staging branch -- only the check schema workflow does. Just because this branch should only be received schema updates. But put here anyway